maintenance_collaboration.Rmd

```diff
@@ -28,18 +31,57 @@ And delete the package current code of conduct if there was one.
 
 We have templates for issue, pull request and contributing guidelines that you can find [in this GitHub repository](https://github.com/ropensci/dotgithubfiles/). There is also [a helper function to insert them into your repository](https://docs.ropensci.org/rodev/reference/use_ro_github.html).
 
-You can tweak them a bit depending on your workflow and package. For example, make sure contributors have instructions in CONTRIBUTING.md for running local tests if not trivial. CONTRIBUTING.md can also contain some details about how you acknowledge contributions (see [this section](#attributions)) and the roadmap of your package (cf [this example](https://github.com/ecohealthalliance/fasterize/blob/master/CONTRIBUTING.md)).
+You can tweak them a bit depending on your workflow and package. 
+
+Example of useful content:
+
+* Style preferences? You might however prefer to make style a configuration (of [lintr](https://github.com/jimhester/lintr), [styler](https://styler.r-lib.org/)) or to [fix code style yourself](https://github.com/rstudio/blogdown/pull/432#pullrequestreview-368391904) especially if you don't use a popular code style like the [tidyverse coding style](https://style.tidyverse.org/).
+
+* Infrastructure like roxygen2?
+
+* Process preferences? Issue before a PR?
+
+* Scope? E.g. in skimr.
+
+* Sandbox account creation? Mocking in tests? Use links to external docs.
+
+Example: [contributing guide template by Peter Desmet](https://gist.github.com/peterdesmet/e90a1b0dc17af6c12daf6e8b2f044e7c).
+
+Adding a personal touch to your contributing guide might make it look less generic and therefore more sincere.
+
+CONTRIBUTING.md can also contain some details about how you acknowledge contributions (see [this section](#attributions)) and the roadmap of your package (cf [this example](https://github.com/ecohealthalliance/fasterize/blob/master/CONTRIBUTING.md)).
 
 We encourage you to direct feedback that is not a bug report or a feature request to [rOpenSci forum](https://discuss.ropensci.org/), after making sure you'd see such questions on the forum. Users can use the forum to ask questions about use and report their use cases, and you can subscribe to individual categories and tags to receive notifications about your package.  Feel free to mention this in the docs of your package and/or the contributing guidelines/issue template.  Please direct your users to tag posts with the package name.
 
 You can use the `lintr` package's own bot, lintr-bot, via continuous integration, to get comments if a commit or PR deteriorates the code style of your package. See [this link for guidance](https://github.com/jimhester/lintr#continuous-integration).
```

Comment:
Sorry, this is not in the parts you changed but it's somewhat related. I'm not sure I would recommend using a bot that posts comments about style. It creates a lot of noise and extra cognitive load. 

The best solution in my opinion is to use the ['pr-commands' action](https://github.com/r-lib/actions/blob/master/examples/pr-commands.yaml). It prevents premature comments / commits (that would result in conflicts) for people that like to work with many small commits and push before everything is ready.

We can discuss this in another issue if you wish.

Reply:
Good point, I'll rephrase/remove!